packages/css/src/components/skip-link/README.md

```diff
@@ -0,0 +1,30 @@
+# Skip Link
+
+Gebruik een Skip link om makkelijk met het toetsenbord naar de belangrijkste inhoud te navigeren.
```

Comment:
We capitalize all words in a multi-word component name to make it easier to recognize – Skip Link.